metabase/metabase
# Add created at to `instance` analytics context

Epic https://github.com/metabase/metabase/issues/18291
Depends on https://github.com/metabase/metabase/pull/18684

This PR adds the date the instance was created at to every Snowplow event.

How to test:
- `docker build -t test-container-image .`
- `docker run -dp 3000:3000 test-container-image`
- Open Metabase
- Make sure that the `pageview` event contains `created_at` field.
